src/mobu/services/flock.py

```diff
@@ -130,6 +134,25 @@ async def stop(self) -> None:
         awaits = [m.stop() for m in self._monkeys.values()]
         await asyncio.gather(*awaits)
 
+    def signal_refresh(self) -> None:
+        """Signal all the monkeys to refresh their busniess."""
+        self._logger.info("Signaling monkeys to refresh")
+        for monkey in self._monkeys.values():
+            monkey.signal_refresh()
+
+    def uses_repo(self, repo_url: str, repo_branch: str) -> bool:
+        match self._config:
+            case FlockConfig(
+                business=NotebookRunnerConfig(
+                    options=NotebookRunnerOptions(
+                        repo_url=url,
+                        repo_branch=branch,
+                    )
+                )
+            ) if (url, branch) == (repo_url, repo_branch):
```

Comment:
Do you need this `if` condition? I thought you could just say `repo_url=repo_url` and `repo_branch=repo_branch` inside the pattern match and the attribute inspection matching would do this for you.

Reply:
That's what I thought too :) That `if` is needed.  The clause in the `match` statement only binds those names :(

Reply:
Oh, yes, indeed, you can only match on constants and enums. Oh well.